src/MainWindow.vala

```diff
@@ -434,11 +406,9 @@ namespace Terminal {
                     color_preferences_dialog = new Dialogs.ColorPreferences (this);
                 }
 
-                color_preferences_dialog.destroy.connect (() => color_preferences_dialog = null);
                 color_preferences_dialog.present ();
             });
 
-            bind_property ("title", header, "title", GLib.BindingFlags.SYNC_CREATE);
```

Comment:
Do we not need this still?

Reply:
Adw.HeaderBar does not have a "title" property - it is replaced by a "title-widget" property and we use that for the binding.